assets/src/blocks/Gallery/getImageTitle.js

```diff
@@ -0,0 +1,13 @@
+export const getImageTitle = (image) => {
+  const caption = image.caption || '';
+  const credits = getCredits(image, caption);
+  return `${caption}  ${credits}`.trim();
+};
+
+const getCredits = (image, caption) => {
```

Comment:
That's another way to make it readable :+1: If you want you can include parts of the suggestion I commented right before you pushed this :smile: https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/613#discussion_r652566019

Reply:
Though those may not be needed now it's isolated, either way works for me :+1: